components/SubscribeSection.vue

```diff
@@ -1,34 +1,17 @@
 <template>
   <div
     ref="containerRef"
-    class="w-full border mx-auto py-8 flex items-center flex-wrap"
-    :class="
-      size === 'lg'
-        ? 'px-8 flex-row justify-between space-x-4'
-        : 'px-4 flex-col justify-start'
-    "
+    class="w-full border mx-auto px-8 sm:px-12 py-8 sm:py-12 flex flex-wrap bg-indigo-700 rounded-3xl flex-col justify-start"
   >
-    <div
-      class="flex flex-col justify-start"
-      :class="size === 'lg' ? 'items-start' : 'items-center'"
-    >
-      <h2
-        class="font-extrabold text-gray-900"
-        :class="size === 'lg' ? 'text-4xl' : 'text-3xl'"
-      >
-        {{ $t("subscribe.get-database-insight") }} 💡
-      </h2>
-      <p class="mt-3 text-lg leading-8 text-gray-500">
+    <div class="flex flex-col justify-start text-white">
```

Comment:
This will fix the text overflow problem.

```suggestion
    <div class="w-full flex flex-col justify-start text-white">
```

Reply:
CSS is hard